src/torrent.cpp

```diff
@@ -1384,16 +1383,87 @@ bool is_downloading_state(int const st)
 #endif
 
 #ifdef TORRENT_SSL_PEERS
+	void torrent::init_ssl(string_view cert)
```

Comment:
it looks like `torrent::init_ssl()` moved up, so it no longer lines up with where the previous function was. This makes it a bit more difficult to see what changed in here. Would you mind moving it down to where it currently is in `master`?

Reply:
i.e. move `torrent::init_ssl()` down right after `torrent::verify_peer_cert`

Reply:
Oh yes sorry!